### DISCUSSION TITLE
Global Star Rating Logo

### DISCUSSION THREAD

**Discussion by raedweb** [TDRA-ae/aegov-dls#30] [General]
Dear Folks,

Can we know the guide of the Global Star Rating Logo location on websites and mobile? means if there are a specific places to be added on all digital media

**Comment by moiz-conceptlz** [comment]
Hello @raedweb 

GSR must be placed in the header - always next to the logo. 

Refer to https://designsystem.gov.ae/docs/blocks/header, which shows how to place the GSR logo. The header block is coded to be responsive and handles the resize values for GSR too.

Hope this is helpful.

**Comment by moiz-conceptlz** [comment]
To add here, GSR logo should be clickable, that opens a modal with a larger version of the GSR logo and contextual information about the rating given to the entity.
